praw/models/auth.py

```diff
@@ -6,6 +6,8 @@
     session,
 )
 
+from typing import Dict, List, NoReturn, Optional, Set, Union
```

Comment:
Correct the import order please. These imports go in their own system level section above prawcore (third party import).

Reply:
Ok